scss/typography/_helpers.scss

```diff
@@ -38,6 +38,29 @@ $subheader-margin-bottom: 0.5rem !default;
 /// @type Number
 $stat-font-size: 2.5rem !default;
 
+/// @dropcaps
+
+// Line height for all the text within dropcaps, disabling default line height's
+$dropcaps-line-height : 1.45 !default;
+
+// @dropcaps-first-letter
+
+// Lets set margin first, used tiny negative margins for proper alignment
+$dropcaps-first-letter-margin-top: 0.02em !default;
+$dropcaps-first-letter-margin-bottom: -0.05em !default;
+
+// Padding Right to the dropcaps first letter to create gap b/w rest of text 
+$dropcaps-first-letter-padding-right: 0.05em !default;
```

Comment:
You do not have to provide so much options. Components should be customizable, yes, but it is useless when options are magic numbers that folks cannot easily understand.

I made some search about how dropcaps in CSS are made and found this: http://code.stephenmorley.org/html-and-css/creating-drop-caps/

For now, we can assume these values will not change:
```css
font-size: 5.2em;
line-height: .5;
margin: 0.192em 0.0962em 0 0;
```

Later, we can add understable options like:
- `$dropcaps-lines`: Number of lines the height of the first letter should be.
- `$dropcaps-space`: Both vertical and horizontal space between the first letter and the paragraph.
But it would require some maths.